### PR TITLE
onObjectCollision* events

### PR DESCRIPTION
**Pull request by Benjamin-Dobell**
Documentation for onObjectCollision* events. Wouldn't mind an extra pair of eyes.

### CODE REVIEW

**Comment by Zehir** [comment]
Fixed conflict

**Comment by Benjamin-Dobell** [comment]
Documents collision events/methods referenced in #15

File: docs/event.md [hunk 1]
```diff
@@ -164,6 +167,100 @@ end
...
 
+###onObjectCollisionEnter(...)
+
+This function is called when an Object starts colliding with a [collision registered](object.md#registercollisions) Object.
+
+!!!info "onObjectCollisionEnter(registered_object, collision_info)"
+	* [<span class="tag obj"></span>](types.md)&nbsp;**registered_object**: The object registered to receive collision events.
+	* [<span class="tag tab"></span>](types.md)&nbsp;**collision_info**: A table containing data about the collision.
+		* [<span class="tag obj"></span>](types.md)&nbsp;**collision_info.*collision_object***: Object coming into contact with `registered_object`.
+		* [<span class="tag tab"></span>](types.md)&nbsp;**collision_info.*contact_points***: Table/array full of contact points, where each 3D point is represented by a (number indexed) table, <em>not</em> a [Vector](vector.md).
```
Comment (Benjamin-Dobell) [inline docs/event.md:178]: Well. this is silly of me. There's no reason for me to use `<em>` when I could use `_`. I'm going to blame the presence of HTML on this line throwing me. Will force push a fix shortly.

Comment (Zehir) [reply docs/event.md]: There is lot of html for formatting, maybe we should fix some css instead.

**Comment by Benjamin-Dobell** [comment]
What's still outstanding for this PR?

I mean, I _could_ merge this myself... but you know 😉

File: docs/event.md [hunk 2]
```diff
@@ -14,6 +14,9 @@ onChat([<span class="tag str"></span>](types.md)&nbsp;message, [<span class="tag
...
 onLoad([<span class="tag str"></span>](types.md)&nbsp;save_state) | Called when a game save is finished loading every Object. It is where most setup code will go. | [<span class="i"></span>](#onload)
+onObjectCollisionEnter([<span class="tag obj"></span>](types.md)&nbsp;registered_object, [<span class="tag tab"></span>](types.md)&nbsp;collision_info) | Called when an Object starts colliding with a [collision registered](object.md#registercollisions) Object. | [<span class="i"></span>](#onobjectcollisionenter)
+onObjectCollisionExit([<span class="tag obj"></span>](types.md)&nbsp;registered_object, [<span class="tag tab"></span>](types.md)&nbsp;collision_info) | Called when an Object stops colliding with a [collision registered](object.md#registercollisions) Object. | [<span class="i"></span>](#onobjectcollisionexit)
+onObjectCollisionStay([<span class="tag obj"></span>](types.md)&nbsp;registered_object, [<span class="tag tab"></span>](types.md)&nbsp;collision_info) | Called **every frame** that an Object starts colliding with a [collision registered](object.md#registercollisions) Object. | [<span class="i"></span>](#onobjectcollisionstay)
```
Comment (omniraptorr) [inline docs/event.md:19]: maybe change to "every frame after an Object starts colliding" ?

Comment (Benjamin-Dobell) [reply docs/event.md]: Should be addressed now.

**Pull request merged**